botbot_plugins/base.py

```diff
@@ -176,9 +176,9 @@ def register(self, plugin):
                 if router_name in [r.name for _, r in self.routers.iteritems()]:
                     rule = attr.route_rule[1]
 
-                    self.output('Route {}: {} ({}, {})'.format(router_name,
-                                                               plugin.slug, key,
-                                                               rule))
+                    self.output(u'Route {}: {} ({}, {})'.format(router_name,
```

Comment:
This small change is needed to make DummyApp work with routes that have unicode characters in them.